src/images/checkbox-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg  xmlns="http://www.w3.org/2000/svg" width="17" height="12" viewBox="0 0 17 12">
+    <polygon fill="#fff" class="cls-1" points="12.2 0 4.8 7.8 1.7 4.7 0 6.5 4.8 11.3 14 1.7 12.2 0"/>
```

Comment:
I can't remove the fill, because we need to use it with background-image in css.